.github/workflows/python-package.yml

```diff
@@ -33,8 +33,21 @@ jobs:
         # stop the build if there are Python syntax errors or undefined names
         flake8 climt/ --count --show-source --statistics
     - name: Build climt
-      env:
-        CLIMT_ARCH: Linux
       run: |
-        python setup.py build
-
+           if [ "$RUNNER_OS" == "Linux" ]; then
+            CLIMT_ARCH= Linux
+            python setup.py build
+           elif [ "$RUNNER_OS"== "Windows"]; then
+            CLIMT_ARCH= Windows
+            python setup.py build 
+           elif [ "$RUNNER_OS"== "Darwin"]; then
```

Comment:
I think `$RUNNER_OS` for Mac is `MacOS`, not `Darwin`. Please check

Reply:
ok